### DISCUSSION TITLE
Slowing down the network speed for testing

### DISCUSSION THREAD

**Discussion by inoyakaigor** [nodejs/help#4759] [Q&A]
I have a project that under the hood makes a lot of fetch queries to other service. Depend on user's input. Because of some circumstances and that fact the Other service is legacy from mammoth era I have to make the `fetch`es one by one. So for test purposes (cheking connection timeouts etc) I need to slow down (throttle? is it more correct word?) network in my application. Is there a way to do it like Chrome's devtools? Maybe there is special CLI parameter?

**Comment by xkid1** [comment]
Used setTimeout per request in your end point.

**Reply by inoyakaigor** [reply]
I make fetch one by one because of some conditions that can break that `fetch` chain. So your solution doesn't quite fit
